app/models/c2_cancel_attributes.rb

```diff
@@ -0,0 +1,32 @@
+class C2CancelAttributes
+  def initialize(auction)
+    @auction = auction
+  end
+
+  def to_h
+    {
+      status: 'cancelled',
```

Comment:
yeah it was "cancelled" at one time but the 18F content guides say to spell it "canceled" so we made the switch on C2 at some pt. Guess we forgot to update the docs!